SQL/0000-00-02-Modules.sql

```diff
@@ -47,3 +47,5 @@ INSERT INTO modules (Name, Active) VALUES ('survey_accounts', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('timepoint_list', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('user_accounts', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('electrophysiology_browser', 'Y');
+
+ALTER TABLE issues ADD CONSTRAINT `fk_issues_7` FOREIGN KEY (`module`) REFERENCES `modules` (`ID`);
```

Comment:
This doesn't belong in the Modules.sql, it belongs in wherever the 'issues' table is defined in the default schema.

Reply:
cant add a foreign key to a table that doesnt exist

Reply:
I see.. I still don't like it, but you have a point.

Reply:
same here... it's the consequence of having multiple schema files